modules/network/express-route-gateway/.bicep/nested_roleAssignments.bicep

```diff
@@ -78,7 +78,7 @@ var builtInRoleNames = {
   'Windows Admin Center Administrator Login': subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'a6333a3e-0164-44c3-b281-7a577aff287f')
 }
 
-resource applicationGateway 'Microsoft.Network/applicationGateways@2023-04-01' existing = {
+resource applicationGateway 'Microsoft.Network/expressRouteGateways@2023-04-01' existing = {
```

Comment:
```suggestion
resource expressRouteGateway 'Microsoft.Network/expressRouteGateways@2023-04-01' existing = {
```

Reply:
The role assignment resource scope and name is impacted by this too

Reply:
Fixed and templates regenerated